CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
```

Comment:
Remove this empty line, please.

Reply:
Thank you for the feedback-I'm not hitched to any specific format. Just to be clear by keeping the "ABRT" you mean changing the description to:

```Description=ABRT coredumpctl message creator```

or were you referring to simply tacking on "ABRT" to what I originally submitted? I think the latter would make the strings a bit longer than they should be.

Reply:
Yes, the original, please.

Reply:
Just pushed a new commit with requested changes